code/modules/mob/living/carbon/human/human_powers.dm

```diff
@@ -1,6 +1,31 @@
 // These should all be procs, you can add them to humans/subspecies by
 // species.dm's inherent_verbs ~ Z
 
+/mob/living/carbon/human/proc/tie_hair()
+	set name = "Tie Hair"
+	set desc = "Style your hair."
+	set category = "IC"
+
+	if(h_style)
+		var/datum/sprite_accessory/hair/hair_style = hair_styles_list[h_style]
+		var/selected_string
+		if(hair_style.length <= 1)
+			to_chat(src, "<span class ='warning'>Your hair isn't long enough to tie.</span>")
+			return
+		else
+			var/list/datum/sprite_accessory/hair/valid_hairstyles = list()
+			for(var/hair_string in hair_styles_list)
+				var/list/datum/sprite_accessory/hair/test = hair_styles_list[hair_string]
+				if(test.length >= 2 && ("Human" in test.species_allowed))
```

Comment:
I wonder if this is also shit code. Could be `(species.name in test.species_allowed)` So we could really easily add it to Tajarans and the like. :thinking: